packages/landingpage-new/components/explore/resources/resources-card.module.scss

```diff
@@ -0,0 +1,73 @@
+@use './../../../../elements/src/components/base/new-theme' as theme;
+
+@mixin Text($weight, $size, $line-height, $spacing, $color) {
+  font-weight: $weight;
+  font-size: $size;
+  line-height: $line-height;
+  letter-spacing: $spacing;
+  color: $color;
+}
+
+.row {
+  display: flex;
+  flex-wrap: nowrap;
+  overflow-x: auto;
+  overflow-y: hidden;
+  padding: 40px 5px 5px;
+}
+
+.card {
+  position: relative;
+  width: 277px;
+  height: 392px;
```

Comment:
The cards should be responsive. It might be enough to wrap them if the screen gets too small